### PR TITLE
Add a flag that determines if we need to run VERIFY_COHERENCE

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/CoherenceBuild/Program.cs [hunk 1]
```diff
@@ -70,7 +70,8 @@ static int Main(string[] args)
                 }
 
                 var processResult = ReadPackagesToProcess(di, buildBranch.Value());
-                if (!CoherenceVerifier.VerifyAll(processResult))
+                var verifyCoherence = Envirornment.GetEnvironmentVariable("VERIFY_COHERENCE") == "true";
```
Comment (anurse) [inline src/CoherenceBuild/Program.cs]: I'd flip this. We only want to disable it when absolutely necessary.

**Comment by pranavkm** [comment]
:up: :date:

**Comment by anurse** [comment]
:ship: :it: :a: :sa: p

**Comment by Eilon** [comment]
Nice :smile:

**Pull request merged**